code/modules/mob/dead/observer/observer.dm

```diff
@@ -296,7 +296,11 @@ This is the proc mobs get to turn into a ghost. Forked from ghostize due to comp
 	set name = "Ghost"
 	set desc = "Relinquish your life and enter the land of the dead."
 
-	if(stat == DEAD)
+	if(src.health < 0 && src.health > -95.0) //crit people
+		succumb()
+		sleep(5)
```

Comment:
It's to make sure it doesn't continue with the else proc first, which is what happened to me during testing.